great_ape_safe/great_ape_safe.py

```diff
@@ -79,7 +79,7 @@ def take_snapshot(self, tokens):
         df['decimals'].append(18)
         for token in tqdm(tokens):
             try:
-                token = interface.ERC20(token) if type(token) != Contract else token
+                token = Contract(token) if type(token) != Contract else token
```

Comment:
`interface.ERC20` doesnt work, because brownie will then also interpret lp tokens as an ERC20, thus excluding the lp methods.